src/lib/locals.js

```diff
@@ -39,9 +41,13 @@ module.exports = {
     const toggleValue = req.app.get("APP.LANGUAGE_TOGGLE_ENABLED");
     res.locals.showLanguageToggle = toggleValue && toggleValue === "1";
     res.locals.htmlLang = req.i18n.language;
-    res.locals.currentUrl = new URL(
-      req.protocol + "://" + req.get("host") + req.originalUrl,
-    );
+    try {
+      res.locals.currentUrl = new URL(
+        req.protocol + "://" + req.get("host") + req.originalUrl,
+      );
+    } catch (e) {
+      Logger.error("Error constructing url for language toggle", e.message);
```

Comment:
I haven't seen the logger used like this before, in other areas of common express we use `logger.get(PACKAGE_NAME)...` which is as per their [docs](https://github.com/HMPO/hmpo-logger). 

Is there a particular reason you've done it like this?

Reply:
will update